### PR TITLE
Fix issue with removing languages from admin panel

### PR DESCRIPTION
**Pull request by nikolaev-fb**
**Problem**
Not possible to remove any language installed by the language pack.

**Steps to reproduce**
1. Install SwagLanguagePack plugin. Version 3.1.0 or 4.0.0
2. Go to Shopware Settings -> Shop -> Languages
3. Try to remove any language

**Expected result**
Language successfully removed

**Actual Result**
Language is not removed and PHP exception occurs.
```
An exception occurred while executing a query: SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`sw6582`.`swag_language_pack_language`, CONSTRAINT `fk.swag_language_pack_language_language` FOREIGN KEY (`language_id`) REFERENCES `language` (`id`) ON DELETE RESTRICT ON UPDATE CASCADE)
```

**Reason**
table `swag_language_pack_language` uses action `ON DELETE RESTRICT` for the language foreign key. This prevents the system from deleting related records in `swag_language_pack_language` when removing languages.

### CODE REVIEW

File: src/Util/Migration/MigrationHelper.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function createPackLanguageTable(): void
                 CONSTRAINT `fk.swag_language_pack_language_language`
                     FOREIGN KEY (`language_id`)
                     REFERENCES `language` (`id`)
-                    ON DELETE RESTRICT
+                    ON DELETE CASCADE
```
Comment (shyim) [inline src/Util/Migration/MigrationHelper.php:40]: can you add a new migration? changing this will only affect new installations

Comment (nikolaev-fb) [reply src/Util/Migration/MigrationHelper.php]: Hi @shyim 
Thank you for your review. 
I have added migration and reverted changes in the helper class.

**Comment by marcelbrode** [comment]
Would be great if you add an entry in both changelog files. If you're not good with German, I'm happy to help

**Comment by nikolaev-fb** [comment]
@marcelbrode I guess changelog files were removed with the previous PR https://github.com/shopware/SwagLanguagePack/pull/1

**Comment by marcelbrode** [comment]
@nikolaev-fb Ah, just learned about it. Then I have no issues here :smile:

**Pull request merged**